### ISSUE TITLE
what's the lrf in config?

### ISSUE THREAD

**Issue by lucasjinreal** [ultralytics/yolov5#1386]
```
 # https://pytorch.org/docs/stable/_modules/torch/optim/lr_scheduler.html#OneCycleLR
    lf = lambda x: ((1 + math.cos(x * math.pi / epochs)) / 2) * (1 - hyp['lrf']) + hyp['lrf']  # cosine
    scheduler = lr_scheduler.LambdaLR(optimizer, lr_lambda=lf)
```

What's the lrf here? lr maxium? why name as lrf?

**Comment by glenn-jocher** [comment]
You can see these here:
https://github.com/ultralytics/yolov5/blob/c4addd7761a4627f6fa97604ad9f1124f439b774/data/hyp.scratch.yaml#L1-L7

The final learning rate with be hyp['lrf'] * hyp['lr0'] using the cosine LR scheduler. You can see all parameter group learning rates in the logged training info:
https://wandb.ai/glenn-jocher/yolov5_tutorial
<img width="1538" alt="Screenshot 2020-11-13 at 09 18 32" src="https://user-images.githubusercontent.com/26833433/99045208-48650e00-2591-11eb-8e74-d67dc543e332.png">

**Comment by lucasjinreal** [comment]
@glenn-jocher thanks for your reply. I notice that there is a term which is plus `hyp['lrf']` (since it's plus thus lr will bigger than lrf anyway, correct me if i am wrong), isn't that too big for train? Default lrf set to 0.2

**Comment by glenn-jocher** [comment]
lr_lambda function provides a multiple for the LR scheduler, not an addition.

**Comment by sam-a-d** [comment]
@glenn-jocher thanks for your response on **lr0 and lrf**. I believe it is the same in **YOLO v8** as well.

**Comment by glenn-jocher** [comment]
@Samad-sust you're welcome! However, just to clarify: there is no YOLOv8 (as of the time of my writing this). The current latest version of YOLO is v5, which is what our repository Ultralytics YOLOv5 maintains. Please let me know if you have any further questions or concerns related to YOLOv5.
